teams/tidb/membership.json

```
@@ -135,6 +135,7 @@
         "spongedu",
         "tangwz",
         "tsthght",
+	"wddevries",
```

the rest of the file is indented by space

```suggestion
        "wddevries",
```